recipes/qrupdate/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "qrupdate" %}
+{% set version = "1.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://downloads.sourceforge.net/project/{{ name }}/{{ name }}/1.2/{{ name }}-{{ version }}.tar.gz
+  sha256: e2a1c711dc8ebc418e21195833814cb2f84b878b90a2774365f0166402308e08
+  patches:
+    # the whole recipe was copied from:
+    # https://github.com/unlhcc/hcc-conda-recipes/tree/master/recipes/qrupdate/1.1.2
+    # with minor modifications
```

Comment:
We want to keep the recipes under BSD-3-Clause. Can you ask them if they are willing to re-license?

Reply:
Wait, these people copy cf recipes all over the place (e.g. see their octave recipe). It was sort of clear to me it was all allowed. We can just rewrite this recipe (I am already modifying it significantly to fix the test).

What exactly needs to be re-licensed?

Reply:
License at https://github.com/unlhcc/hcc-conda-recipes/blob/master/LICENSE

Reply:
I opened https://github.com/unlhcc/hcc-conda-recipes/issues/12 about the octave recipe

Reply:
Btw, I assumed you didn't copy their octave recipe and they copied it from us --- but I am not sure. I just came across it and I thought it looked way too similar. Did you verify my claim at least?

Reply:
I wrote the octave recipe myself.